source/common/ssl/connection_impl.cc

```diff
@@ -187,6 +188,39 @@ std::string ConnectionImpl::sha256PeerCertificateDigest() {
   return Hex::encode(computed_hash);
 }
 
+std::string ConnectionImpl::uriSanPeerCertificate() {
```

Comment:
Can you confirm via coverage build that all lines in this function have coverage? If not, please add tests so that we have total coverage.